uber/decorators.py

```diff
@@ -105,11 +105,11 @@ def zipfile_out(self, session):
 
 def csv_file(func):
     @wraps(func)
-    def csvout(self, session):
+    def csvout(self, session, *args, **kwargs):
```

Comment:
See below, but you probably only want ``**kwargs`` here and not ``*args`` since CherryPy passes all query parameters as keyword arguments.